.atomist/mocha/SimpleTest.ts

```diff
@@ -1,10 +0,0 @@
-import "mocha";
-import assert = require("power-assert");
-
-describe("default test", () => {
-
-    it("should be true", () => {
-        assert(true === true);
-    });
-
-});
```

Comment:
This was part of the commit @cdupuis wanted to be included: https://github.com/atomist/github-rugs/pull/45/commits/5878994b0013f47762725aa6eee724d14d8146ad  Should I add this back?